docs/index.html

```diff
@@ -32,7 +32,7 @@
         '.*?/awesome': 'https://raw.githubusercontent.com/docsifyjs/awesome-docsify/master/README.md',
         '.*?/changelog': 'https://raw.githubusercontent.com/docsifyjs/docsify/master/CHANGELOG.md',
         '/.*/_navbar.md': '/_navbar.md',
-        '/zh-cn/(.*)': 'https://cdn.jsdelivr.net/gh/docsifyjs/docs-zh/$1',
+        '/zh-cn/(.*)': 'https://cdn.jsdelivr.net/gh/docsifyjs/docs-zh@master/$1',
```

Comment:
why `@` ?

Reply:
@master is the default, plus it is for easy replacement, is there any better implementation? I am not professional.

Reply:
ok lets keep it like this for now. I will take a look later if possible 👍

Reply:
https://github.com/docsifyjs/docsify/blob/2cf6fd8698af916f1d29b76578f8b63d52028c7a/docs/index.html#L83